src/scene/materials/standard-material-options-builder.js

```diff
@@ -287,6 +288,10 @@ class StandardMaterialOptionsBuilder {
         options.litOptions.useIridescence = stdMat.useIridescence && stdMat.iridescence !== 0.0;
         options.litOptions.useMetalness = stdMat.useMetalness;
         options.litOptions.useDynamicRefraction = stdMat.useDynamicRefraction;
+
+        options.litOptions.useDisplacement = stdMat.useDisplacement;
+        options.litOptions.displacementFactor = stdMat.displacementFactor;
+        options.litOptions.displacementOffset = stdMat.displacementOffset;
```

Comment:
these two lines should not be here, we don't need a new shader each time a value changes, which this would cause as options are used to generate hash